docs/24pullrequests.md

```diff
@@ -18,4 +18,4 @@ Something missing?  Create a pull request!
 
 Check out the [Getting Involved in a Track](https://github.com/exercism/exercism.io/blob/master/docs/getting-involved-in-a-track.md) documentation, and some [ideas for things we are looking for](http://exercism.io/languages/ruby/todo) for exercises that are needed to be implemented.
 
-We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/ruby](https://badges.gitter.im/exercism/ruby.svg)](https://gitter.im/exercism/ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
+We are available to answer questions in the  [![Join the chat at https://gitter.im/exercism/ruby](https://badges.gitter.im/exercism/ruby.svg)](https://gitter.im/exercism/ruby?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge).
```

Comment:
I can't tell what the change is here.

Reply:
I can't either, but the two spaces after the word "the" could be fixed.  It is stating that there is a "non-visible" change in that line.  Hex Editor... here we come!

Reply:
It could be a line-ending change, that ends up not being visible, yet still a change.